resources/service_sysvinit.rb

```diff
@@ -0,0 +1,92 @@
+provides :nexus3_service_sysvinit
+provides :nexus3_service, os: 'linux'
+
+property :instance_name, String, name_property: true
+property :install_dir, String
+property :nexus3_user, String
+property :nexus3_group, String
+
+# TODO: easy
+# http://www.sonatype.org/nexus/2017/01/25/how-to-install-latest-sonatype-nexus-3-on-linux/
+# sudo ln -s /app/nexus/bin/nexus /etc/init.d/nexus
+# sudo chkconfig --add nexus
+# sudo chkconfig --levels 345 nexus on
+
+action :start do
+  create_init
+
+  service "nexus3_#{new_resource.instance_name}" do
+    # TODO: define in helper (see Tomcat)
+    provider platform_sysv_init_class
+    supports restart: true, status: true
+    action :start
+  end
+end
+
+action :stop do
+  service "nexus3_#{new_resource.instance_name}" do
+    provider platform_sysv_init_class
+    supports status: true
+    action :stop
+    only_if { ::File.exist?("/etc/init.d/nexus3_#{new_resource.instance_name}") }
+  end
+end
+
+action :restart do
+  service "nexus3_#{new_resource.instance_name}" do
+    provider platform_sysv_init_class
+    supports status: true
+    action :restart
+  end
+end
+
+action :enable do
+  create_init
+
+  service "nexus3_#{new_resource.instance_name}" do
+    provider platform_sysv_init_class
```

Comment:
Mostly I got "copy-n-paste inspiration" from the tomcat cookbook. Maybe they're wrong.